src/components/JsxParser.js

```diff
@@ -45,7 +45,10 @@ export default class JsxParser extends React.Component {
       raw.replace(new RegExp(`(</?)${tag}`, 'ig'), '$1REMOVE')
     , rawJSX).trim()
 
-    const doc = parser.parseFromString(jsx, 'text/html')
+    const wrappedJsx = `<!DOCTYPE html>\n<html><body>${jsx}</body></html>`
```

Comment:
Adding the `DOCTYPE`, `html` and `body` tags in this manner assumes that the fragment doesn't already contain them. It also alters the output by injecting them into it.

If they are necessary in order for the parsing to work correctly, it might be better to first inspect the input, and only add these tags if needed - then strip them at the end, if they were added - so that the total output remains equivalent to the input.